handler/commentHandler.go

```diff
@@ -19,19 +19,21 @@ import (
 )
 
 const (
-	openConstant            string = "open"
-	ClosedConstant          string = "closed"
-	closeConstant           string = "close"
-	reopenConstant          string = "reopen"
-	lockConstant            string = "Lock"
-	unlockConstant          string = "Unlock"
-	setTitleConstant        string = "SetTitle"
-	assignConstant          string = "Assign"
-	unassignConstant        string = "Unassign"
-	removeLabelConstant     string = "RemoveLabel"
-	addLabelConstant        string = "AddLabel"
-	setMilestoneConstant    string = "SetMilestone"
-	removeMilestoneConstant string = "RemoveMilestone"
+	openConstant             string = "open"
+	ClosedConstant           string = "closed"
```

Comment:
Hi, why did this get a capital letter?

Reply:
This is old code, not from this commit. All `sting` values are moved right and that's why it's shown in the diff.

Reply:
It's used in main.go for a check: https://github.com/alexellis/derek/blob/master/main.go#L83

Reply:
This was exported when the handler package was introduced.